recipes/ledger/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ledger" %}
+{% set version = "3.1.3" %}
+{% set sha256 = "b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ledger/ledger/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - fix-build.patch
+
+build:
+  number: 0
+  skip: True  # [win or osx or py>27]
+  run_exports:
+    - {{ pin_subpackage("ledger") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - python
```

Comment:
You need to remove this here as `python` should be in host.

Reply:
Even if ledger requires the Python dev libraries to build?

Reply:
Yes, please read https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build

Reply:
Thanks for linking that, it makes sense now; commit pushed